docs/LogGuide.md

```diff
@@ -1812,6 +1973,11 @@ There are also a number of examples where tethers are generated in some other wa
 - Suzaku Extreme birbs: who knows
 - player to player tethers (dragonsight, cover, fairy tether)
 
+There is currently no log line that indicates that a tether is no longer present.
+Some mechanics may periodically "re-apply" the tether, resulting in multiple redundant lines.
+If a tether changes (for example, a tether which players must stretch out),
+it generates a new log line.
+
```

Comment:
Unrelated to this PR/update, but there is a `TetherCancel` (`47`/`0x2F`) ActorControl packet that we could emit from OP if needed? They don't ever contain the tether ID so I'm not sure how useful exactly they'd be.

Here's an example from UCOB the other day (with my network packet debug dump showing the AC packets):

```
35|2024-04-25T23:24:43.7730000-04:00|4001851B|Firehorn|10001234|Some One|0000|0000|0005|10001234|000F|0000|b9cf98cc381ad7dc
4096|2024-04-25T23:24:43.7730000-04:00|ActorControl|4001851B|Tether|00000000|00000005|10001234|0000000F|00000000|a20ea866e59b57bb
4096|2024-04-25T23:24:48.7720000-04:00|ActorControl|4001851B|TetherCancel|00000000|00000000|00000000|00000000|00000000|6a3deb59597544fe
```

Reply:
Have you looked into how tethers are represented in memory? I wonder if it's the case that a given pair of entities can only have one tether, so you know exactly what tether is being canceled.

Reply:
I looked into it I think 8 or 9 months ago and it was a mess, something about tethers being controlled by the director and not directly associated with the actor? I can't remember all the details off the top of my head.

I'm not sure what limitations, if any, are associated with the tether logic. It might be that the field following the actor ID (`0xF` in my example above) is an index of some sort, or flags, or something else.

`param2` is sometimes `0xA` in the `TetherCancel` AC packet, but only when the source ID is a player actor, and these TetherCancel ACs never have an associated `Tether` AC packet and the tether field on their `SpawnPlayer` packet is null, e.g.:
```
4096|2024-04-12T22:47:09.7790000-04:00|ActorControl|10001234|TetherCancel|00000000|0000000A|00000000|00000000|00000000|2a018d79676a8d04
```